.github/workflows/build-and-push.yml

```diff
@@ -53,8 +52,8 @@ jobs:
       - name: Configure AWS credentials
         uses: aws-actions/configure-aws-credentials@v1
         with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          role-to-assume: arn:aws:iam::776473272850:role/dev-github-oidc-deploy
```

Comment:
We could probably lock this down further at some point to just a role that has e.g. ECR full access. Not now though 🙂